test/java/com/arpnetworking/metrics/portal/integration/repositories/JobExecutionRepositoryIT.java

```diff
@@ -174,6 +175,22 @@ public Integer visit(final JobExecution.Started<T> state) {
         }).apply(lastRun.get());
     }
 
+    @Test
+    public void testJobScheduledInThePast() {
```

Comment:
This test catches the false positives from using the current date (since a partition will exist for that regardless of if `DailyPartitionCreator` succeeds.